examples/opencv/edges.json

```diff
@@ -10,6 +10,7 @@
   },
   "transform": {
     "cmd": [ "python3", "/edges.py" ],
-    "image": "pachyderm/opencv"
+    "image": "pachyderm/opencv",
+    "dockerfile": "./Dockerfile"
```

Comment:
I think it should error if you specify both of these, since it's ambiguous where the image should be coming from. We probably want to keep our examples referencing images rather than Dockerfiles, particularly this one because the opencv container takes a while to build.

Reply:
This might be lack of familiarity with k8s, but if you specified a dockerfile, you'd still need to push an image to docker hub, no? Otherwise where would k8s fetch the image? I figured directly pushing the image from docker to k8s once it's built would be finicky.

Reply:
Right, I forgot that even after you build the image you still need something to tag it as. We could just default to using the pipeline's name, but having the image in there seems to make sense as well.

Also it just occurred to me that if someone does `--build` without setting `"dockerfile"` we should default to `Dockerfile` in the current directory, since that's almost always what it's going to be.

Reply:
Is having multiple pipeline specs in a single directory atypical? It'd break those, including this opencv example (i.e. if you ran build on montage.)

Reply:
Having multiple specs in the same directory is pretty common I think, especially because we support having multiple specs in the same file. But having multiple specs using the same docker image / dockerfile (with different commands) is also pretty common. So I think the default is still going to make sense in a lot of situations.